### ISSUE TITLE
Project Licence

### ISSUE THREAD

**Issue by kevihiiin** [Mischback/django-project-skeleton#6]
Hi @Mischback,

I want to use the code generated by your skeleton for another open source project. The project will be licensed under GPLv3, given that I could not find a licence on your repository I wanted to ask if this would be compatible and if you are planning to publish the project under a license.

**Comment by Mischback** [comment]
Hi @kevihiiin, 

funnily I'm currently working on a massive new feature-set and have the topic *licence* on my todo list.

**tl;dr:** Next version of **django-project-skeleton** will include **MIT License**.


Just to clarify, **why** there currently is no licence included:
The **django-project-skeleton** is, as the name suggests, a skeleton, that can and should be used to create and maintain Django-based web applications. Some files (e.g. `manage.py` or `wsgi.py`) are slightly modified versions of the files provided by Django itsself.

When I created **django-project-skeleton** I was not aware, that omitting a licence could possibly hamper its usage. My thought was, that the template should not provide a licence, because I didn't want to restrict the users to a given licence.

Once https://github.com/Mischback/django-project-skeleton/pull/5 is finished, it will include a licence file, that will get replaced/deleted during an initialisation step (most likely a `make init`).

I plan to update the repository to be available under **MIT License**. I'm not a lawyer, but I think that it should be compatible with GPLv3.

Please drop me another message, I you / your team requires the licence file to be available now, I will provide a *hotfix* commit to `development` / `master`.
